cellprofiler/gui/cpframe.py

```diff
@@ -972,113 +972,119 @@ def __on_help_path_list(self, event):
-Group found at https://github.com/aseprite/aseprite/blob/master/docs/
-licenses/libjpeg-LICENSE.txt; (d) matplotlib © 2003-2012 John Hunter,
-© 2012-2013 Michael Droettboom et al. under licenses found at
-http://matplotlib.org/users/license.html#copyright-policy; (e) Java
-Runtime Environment © 2013 Oracle, Inc. under the Oracle Binary Code
-License Agreement for the Java SE Platform Products and JavaFX found
-at http://www.oracle.com/technetwork/java/javase/terms/license/
-index.html; (f) Bioformats © 2000-2013 University of Dundee & Open
-Microscopy Environment; ice © 2013 ZeroC, Inc.; mySQLdb © 2013 Andy
-Dustman; omero © 2002-2013 Open Microscopy Environment; and ZeroMQ ©
-2007-2012 iMatix Corporation and Contributors; (g) wxWidgets ©
-1999-2005 Julian Smart, et al.; and wxPython © 1992-2006 Julian Smart,
-et al. and © 1996 Artificial Intelligence Applications Institute under
-the wxWindows license; (h) Intel Visual Fortran Compiler Run-time
-Library, Intel Fortran Portability Library, Intel OMP Runtime Library,
-Math Library for Intel Compilers, Short Vector Math Library for Intel
-Compilers under the Single User license provision for
-Resdistributables © 2013 Intel Corporation under the End User License
-Agreement for the Intel Software Development Products; and
-(i) FastEMD © Copyright (c) 2009-2012, Ofir Pele. under BSD licenses
-
-"""
-
-        icon_copyrights = get_icon_copyrights()
-        if additional_copyrights is not None:
-            additional_copyrights += icon_copyrights
-        button_list.append(("Additional licenses and attributions", 
-                            additional_copyrights))
-
-        for button_text, license_text in button_list:
-            sub_sizer = wx.BoxSizer(wx.HORIZONTAL)
-            sizer.Add(sub_sizer, 0, wx.EXPAND | wx.ALL, 5)
-            license_button = wx.Button(dlg, -1, button_text + " >>")
-            sub_sizer.Add(license_button, 0, wx.ALIGN_LEFT | wx.RIGHT, 3)
-            sub_sizer.Add(wx.StaticLine(dlg), 0, wx.ALIGN_CENTER_VERTICAL | wx.EXPAND)
-
+        with wx.Dialog(self, size=(640, 800)) as dlg:
+            dlg.Title = "About CellProfiler"
+            sizer = wx.BoxSizer(wx.VERTICAL)
+            dlg.SetSizer(sizer)
             panel = wx.lib.scrolledpanel.ScrolledPanel(dlg)
-            panel.Sizer = wx.BoxSizer(wx.VERTICAL)
-            license_ctl = wx.StaticText(panel, -1, license_text)
-            panel.SetMinSize((license_ctl.Size[0] + 20, 320))
-            panel.Sizer.Add(license_ctl, 1, 
-                            wx.EXPAND | wx.ALL, 2)
-            sizer.Add(panel, 0, wx.EXPAND | wx.ALL, 5)
+            panel.SetAutoLayout(1)
+            sizer.Add(panel, 1, wx.EXPAND)
+            sizer = wx.BoxSizer(wx.VERTICAL)
+            panel.SetSizer(sizer)
+            sizer.Add(
+                wx.StaticBitmap(panel, -1, splashbitmap), 0, wx.EXPAND | wx.ALL, 5)
+            tcstyle = wx.TE_READONLY | wx.BORDER_NONE
```

Comment:
Anne asked to be able to cut and paste the version text, so I used a read-only text box instead of a static control here - it's an old trick.